contrib/utilities/indent_common.sh

```diff
@@ -43,7 +43,7 @@ checks() {
   export PATH="${CLANG_FORMAT_PATH}:${PATH}"
 
   if ! [ -x "$(command -v "${DEAL_II_CLANG_FORMAT}")" ]; then
-    echo "***   No clang-format program found."
+    echo "***   No clang-format program found, or found in the wrong version."
```

Comment:
```suggestion
    echo "***   No clang-format program found, or found with the wrong version."
```
?

Reply:
@peterrum  I think both are correct, but I'm happy to go with your suggestion -- so implemented!